opentracing/span.py

```diff
@@ -175,3 +162,10 @@ def get_trace_attribute(self, key):
         :return: value of the Trace Attribute with given key, or None.
         """
         return None
+
+    def tracer(self):
```

Comment:
the most efficient way would be exposing `tracer` attributes directly as public field. Since it makes it r/w, the safer way would be a method like here, but then I would go for a `@property`, to preserve the `span.tracer` syntax.